ledger-core/v2/pulse/pulse_number.go

```diff
@@ -38,7 +38,7 @@ const (
 	MaxTimePulse = 1<<30 - 1
 
 	// PulseNumberSize declares the number of bytes in the pulse number
-	NumberSize int = 4
+	NumberSize = 4
```

Comment:
NumberSize is pulse.Number? :O

Reply:
no, it is untyped int constant
try this - there will be no errors
	i := 0
	if i == NumberSize {
		
	}

Reply:
OK, my bad :) Somehow I was thinking that it'll adopt type from previous declarations, but that is only in case of iota